server/webapp/WEB-INF/rails.new/spec/controllers/api_v1/server_version_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+##########################################################################
+# Copyright 2016 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################################
+
+require 'spec_helper'
+
+describe ApiV1::ServerVersionController do
+  describe :show do
+    it 'should render the current gocd server version for admins' do
+      version_file = Rails.root.join('..', 'vm', 'admin', 'server_version.txt.vm')
+      file_content = '{"go_version":"16.6.0", "go_build_number":"235", "git_sha":"69ef4921709a84831913d9fa7e750fbf840f213c"}'
+      expected_json = { :_links => {
+                          :self=>{:href=> 'http://test.host/api/server_version' },
```

Comment:
Sorry, I mean this https://github.com/gocd/gocd/blob/1236015adabbfdf88a786f9e0d66aabd071c1c12/server/webapp/WEB-INF/rails.new/spec/controllers/api_v1/users_controller_spec.rb#L33